### PR TITLE
Feature/satingest.v4.5.0

### PR DESCRIPTION
**Pull request by SteveStegall-NOAA**

### CODE REVIEW

File: fix/aerosol.af_viirs [hunk 1]
```diff
@@ -8,8 +8,12 @@ echo " Starting $TASK and  $TASK.$family "
 
       eval ORBITLIST${n}=af_viirs
       eval REMOTEDSNGRP${n}='"concatenate_families \
-PDAFileLinks/PULL/ASCII/AF_v1r2_npp_s???????????????_e???????????????_c???????????????.txt \
-PDAFileLinks/PULL/ASCII/AF_v1r2_j01_s???????????????_e???????????????_c???????????????.txt"'
+      eval REMOTEDSNGRP${n}='"concatenate_families \
```
Comment (ilianagenkova) [inline fix/aerosol.af_viirs:11]: Pls, remove line 10 or 11

File: fix/poes_sst.navy_avhrr [hunk 2]
```diff
@@ -15,9 +15,8 @@ echo " Starting $TASK and  $TASK.$family "
       eval REMOTEDSNGRP${n}=\${REMOTEDSNGRP${n}:-'"concatenate_families \
 mcsst_mtc_d*tmp \
 mcsst_mtb_d*tmp \
-mcsst_mta_d*tmp"'}
-#mcsst_npp_d*tmp \
-#mcsst_n20_d*tmp \
+mcsst_npp_d*tmp \
```
Comment (ilianagenkova) [inline fix/poes_sst.navy_avhrr:18]: When you tested this, did you save the b012/xx012 tanks with extra data? Path?

**Review by ilianagenkova — CHANGES_REQUESTED** [review]
@SteveStegall-NOAA, good work so far! One small change needed.

File: fix/poes_sst.navy_avhrr (hunk 2 above)
Comment (SteveStegall-NOAA) [reply fix/poes_sst.navy_avhrr:18]: I am testing this now.  Will let you know when it is done and point the dir to you.  Thanks.

Comment (SteveStegall-NOAA) [reply fix/poes_sst.navy_avhrr:18]: Hello,  Here is where the tank is for the test ingest:
[clogin02 /lfs/h2/emc/obsproc/noscrub/steve.stegall/DCOMDIR/RB_4.5_test/prod/20230104/b012]$ ll
total 5001076
-rw-r--r-- 1 steve.stegall obsproc 2559120464 Jan  4 17:05 xx012
-rw-r--r-- 1 steve.stegall obsproc 2486921832 Jan  4 17:32 xx023
-rw-r--r-- 1 steve.stegall obsproc   74967016 Jan  4 17:06 xx222

Comment (ilianagenkova) [reply fix/poes_sst.navy_avhrr:18]: Thanks! I'll wait for the user to chime in on their needs...

**Pull request merged**